facia-tool/public/css/style.css

```diff
@@ -1533,9 +1536,9 @@ button {
     width: 120px;
 }
 
-.latest-articles .tool--small--copy,
-.latest-articles .tool--small--copy.large {
-    width: 83px;
+.latest-articles .tool--small--copy.large,
+.latest-articles .tool--small--copy-to-clipboard.large {
```

Comment:
This is interesting. You have '.tool--small--copy.large' We are creating object which is called 'small' and making it large by adding class 'large' :-)

This would be probably fixed with the styles I've mentioned above. You would be able to do this class="tool tool--large tool--copy"